src/webvr-manager.js

```diff
@@ -329,16 +336,30 @@ WebVRManager.prototype.anyModeToNormal_ = function() {
 
 WebVRManager.prototype.resizeIfNeeded_ = function(camera) {
   // Only resize the canvas if it needs to be resized.
-  var size = this.renderer.getSize();
-  if (size.width != window.innerWidth || size.height != window.innerHeight) {
-    camera.aspect = window.innerWidth / window.innerHeight;
+  var size = this.renderer.getSize(),
+      elem = this.getParentElem_(),
+      width = typeof elem.innerWidth === 'number' ? elem.innerWidth : elem.offsetWidth,
+      height = typeof elem.innerHeight === 'number' ? elem.innerHeight : elem.offsetHeight;
+
+  if (size.width != width || size.height != height) {
+    camera.aspect = width / height;
     camera.updateProjectionMatrix();
-    this.resize_()
+    this.resize_();
   }
 };
 
+WebVRManager.prototype.getParentElem_ = function () {
+  var elem = this.parentElem;
+  
+  return elem && elem instanceof Element ? elem : window;
```

Comment:
I'm not totally satisfied with this myself. If you want to continue using a constructor-defined reference to the element, I agree with you.